### PR TITLE
Add abstain rev acct

### PR DESCRIPTION
**Pull request by jimscarver**
Add abstain rev acct for when someone changes their vote to abstain.

does not vote abstain unless vote is changed back to abstain so total abstain votes cannot be counted and may not be needed. support and oppose can be counted and the rest assumed to be abstain.

### CODE REVIEW

File: src/web/ballot.js [hunk 1]
```diff
@@ -372,7 +372,7 @@ function renderQuestion(id, qix, { shortDesc, docLink, yesAddr, noAddr }, { rend
     const radio = (value, props = {}) =>
         check.theInput(render('input', { type: 'radio', name, value, title: value, ...props }));
 
-    const answers = [radio(noAddr), radio('', {checked: 'checked'}), radio(yesAddr)];
+    const answers = [radio(noAddr), radio(abstainAddr, {checked: 'checked'}), radio(yesAddr)]; //radio('', {checked: 'checked'})
```
Comment (dckc) [inline src/web/ballot.js:375]: don't comment out old code; delete it.

File: package.json [hunk 2]
```diff
@@ -39,6 +39,7 @@
     "plugins": []
   },
   "dependencies": {
+    "@tgrospic/rnode-client-js": "file:.yalc/@tgrospic/rnode-client-js",
```
Comment (dckc) [inline package.json:42]: why is this here? The current design is to use a submodule; does that not suffice?

Does it work? Hm... netlify is not testing this PR because it targets the `progress-tour` branch. I wonder if it's a good idea to target that branch. I'm a little swapped out.

File: public/index.html [hunk 3]
```diff
@@ -41,7 +41,7 @@
     <div class="app container">
       <div class="testNet">
         <div class="jumbotron">
-          <h1>RChain Annual Meeting 2020 Ballot <sup>(Beta Test)</sup></h1>
+          <h1>RChain Annual Meeting 2020 Ballot <sup>(Beta Test2)</sup></h1>
```
Comment (dckc) [inline public/index.html:44]: Is this change on purpose?

File: src/web/ballot.js [hunk 4]
```diff
@@ -392,7 +392,7 @@ function logged(label, x) {
 }
 
 /**
- * @typedef {{ shortDesc: string, docLink?: string, yesAddr: string, noAddr: string }} QInfo
+ * @typedef {{ shortDesc: string, docLink?: string, yesAddr: string, noAddr: string, abstainAddr: string  }} QInfo
```
Comment (dckc) [inline src/web/ballot.js:395]: nice work!

**Review by dckc — CHANGES_REQUESTED** [review]
I doubt that the change to package.json works.

**Comment by dckc** [comment]
I amended dfb2296 to take out the change to `package.json` and then merged it: c439017870c7

File: package.json (hunk 2 above)
Comment (jimscarver) [reply package.json:42]: that did work but we just took out the .yalc reference which also worked.
    "rnode-client-js": "@tgrospic/rnode-client-js",

File: public/index.html (hunk 3 above)
Comment (jimscarver) [reply public/index.html:44]: steve did that to see a change on the page. it is not needed.

File: src/web/ballot.js (hunk 4 above)
Comment (jimscarver) [reply src/web/ballot.js:395]: thank @TheoXD

**Comment by jimscarver** [comment]
the package.json references yalc, we did not need that

**Comment by jimscarver** [comment]
the package.json references yalc, we did not need that
